### PR TITLE
BCMOHAD-17896 || Updated the formula

### PR DESCRIPTION
**Pull request by RanadheerRG**
# Description

Updated the Report field formula: IND_Q05A:
"If, yes Indigenous heritage" contains "First Nations" then cell is "1" OR, if No, then cell is "2" OR, if blank, cell is "0"

Fixes # (issue)

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

# List high-level changes as bullet items

Below are some examples

- [ ] changed field IND_Q05A formula

# Deployment Tracker

MAiD - Case Management App/force-app/main/default/objects/S1_HC_2023__c/fields/IND_Q05A__c.field-meta.xml

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: MAiD - Case Management App/force-app/main/default/objects/S1_HC_2023__c/fields/IND_Q05A__c.field-meta.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
     <fullName>IND_Q05A__c</fullName>
+    <deprecated>false</deprecated>
     <externalId>false</externalId>
-    <formula>IF( ISBLANK(Form_1634__r.Case__r.If_yes_Indigenious_heritage__c) , 0, IF( INCLUDES(Form_1634__r.Case__r.If_yes_Indigenious_heritage__c, &apos;First Nations&apos;) , 1, 2))</formula>
-    <formulaTreatBlanksAs>BlankAsZero</formulaTreatBlanksAs>
+    <formula>IF( ISBLANK(Form_1634__r.Case__r.If_yes_Indigenious_heritage__c) , 0, IF( INCLUDES(Form_1634__r.Case__r.If_yes_Indigenious_heritage__c, &quot;First Nations&quot;) , 1, 2))</formula>
```
Comment (cgijeffolsen) [inline MAiD - Case Management App/force-app/main/default/objects/S1_HC_2023__c/fields/IND_Q05A__c.field-meta.xml:6]: "If, yes Indigenous heritage" contains "First Nations" then cell is "1" OR, if No, then cell is "2" OR, if blank, cell is "0"

Actual Function:
- Where If_yes_Indigenious_heritage__c is blank it returns 0
- If_yes_Indigenious_heritage__c containing "First Nations" returns 1
- Default response when not blank and not "First Nations" returns 2

My reading of this is as follows:
- Blank is not First Nations (0)
- Unspecified First Nations where just First Nations is entered is (1)
- Any other specific First Nations entry returns (2)

Function is performing as expected. @RanadheerRG

**Review by cgijeffolsen — APPROVED** [review]
Change is specific to use case and functioning as expected. @RanadheerRG Approved

**Comment by cgijeffolsen** [comment]
@RanadheerRG corrected the issue where <formulaTreatBlanksAs>BlankAsZero</formulaTreatBlanksAs> would have been interfering with the results of the decision.

**Review by phsaSunil — APPROVED** [review]

**Review by NataliaNikishina — APPROVED** [review]
@RanadheerRG approved.

**Pull request merged**